deps/foras.ts

```diff
@@ -0,0 +1,4 @@
+import { Foras } from "https://deno.land/x/foras@v2.1.4/src/deno/mod.ts";
```

Comment:
Did you try this? https://docs.deno.com/api/web/~/CompressionStream
It's a web standard supported by Deno. If it outputs the same result, we don't need a dependency.

Reply:
I have seen it but since you cannot pass options to it (level, etc.) I used https://deno.land/x/foras@v2.1.4 where you have more granular control of how it's compressed.

There is also `node:zlib` which also allows you to use gzip with options. However I wanted to avoid node libs since lume does not use any as of now.

However, I am absolutely fine with `CompressionStream` too. If you want me to, I would happily update the PR to use `CompressionStream`.

Reply:
I like the plugins that "just works", with minimum configuration as possible. If you think the compression level is an important thing that many people like to adjust, the dependency is okay. But if you think the CompressionStream is good enough for most cases, I'd use it.